schema/revert/application_validation_functions/emission_total_matches_rev_zero.sql

```diff
@@ -0,0 +1,7 @@
+-- Revert ggircs-portal:application_validation_functions/emission_total_matches_rev_zero from pg
+
+begin;
+
+drop function ggircs_portal.emission_total_matches_rev_zero;
+
```

Comment:
this revert function should also delete the row inserted in the deploy file. Otherwise an incremental revert/deploy will fail because of the unique index on validation_function_name.

Reply:
@pbastia I can fix this.